src/main/java/com/codingnomads/betty/data/batch/footballmatchjob/FootballMatchApiReader.java

```diff
@@ -16,20 +16,20 @@
 @Component
 public class FootballMatchApiReader implements ItemReader<List<FootballMatchInfo>> {
 
-    private GameInformationService gameInformationService;
+    private FootballMatchInfoService footballMatchInfoService;
     private boolean readState = false;
 
     @Autowired
-    public FootballMatchApiReader(GameInformationService gameInformationService) {
-        this.gameInformationService = gameInformationService;
+    public FootballMatchApiReader(FootballMatchInfoService footballMatchInfoService) {
+        this.footballMatchInfoService = footballMatchInfoService;
     }
 
     @Override
     public List<FootballMatchInfo> read() throws Exception, UnexpectedInputException, ParseException, NonTransientResourceException {
 
         if(!readState){
             readState = true;
```

Comment:
I still will approve but given the previous batch job errors we have seen, it is probably a good idea to add a custom exception here

Reply:
ah yes you are right we deserve to see a message:D